pybossa/jobs.py

```diff
@@ -380,19 +386,23 @@ def get_non_contributors_users_jobs(queue='quaterly'):
     for row in results:
         user = User.query.get(row.id)
 
-        subject = "Why don't you help us?!"
-        body = render_template('/account/email/noncontributors.md',
-                               user=user.dictize(),
-                               config=current_app.config)
-        mail_dict = dict(recipients=[user.email_addr],
-                         subject=subject,
-                         body=body,
-                         html=misaka.render(body))
-
-        job = dict(name=send_mail,
-                   args=[mail_dict],
-                   kwargs={},
-                   timeout=(10 * MINUTE),
-                   queue=queue)
-        jobs.append(job)
+        if user.subscribed:
```

Comment:
I was thinking the same, but it will imply a join :-) So I decided to keep things simple and fast, as all the user_ids come from one table and it will save time ;-)